src/quo2/components/graph/wallet_graph/view.cljs

```diff
@@ -18,17 +18,26 @@
     :1-year   365
     500))
 
+(defn- get-line-color
+  [customization-color state theme]
+  (if customization-color
```

Comment:
how about `cond` to flatten it out ?

oh and btw with `success` and `danger` we should be using these as keywords imo and resolve with custom-color function. 

```
(defn- get-line-color
  [customization-color state theme]
  (let [color-keyword (cond
                        customization-color customization-color
                        (= state :positive) :success
                        :else :danger)]
    (colors/theme-colors
     (colors/custom-color color-keyword 50)
     (colors/custom-color color-keyword 60)
     theme)))
```
🤔